.github/workflows/build.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   build-amd64:
     name: Build AMD64 binaries
-    runs-on: longhorn-infra-amd64-runners
+    runs-on: longhorn-infra-oracle-arm64-runners
```

Comment:
Create a new one for amd64? Or just use a general runner from CNCF?

In the current infra, each runner is a pod/container, so 3 node infra cluster can run several runners. We might need to setup some similar on the new infra later.

Reply:
For SPDK, we need a customized runner for both arm64 and amd64 because of the necessary kernel modules and hugepages.

Reply:
> In the current infra, each runner is a pod/container, so 3 node infra cluster can run several runners. We might need to setup some similar on the new infra later.

Yes, we do the same thing in the oracle runner by https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/about-actions-runner-controller

Reply:
Did you install a single node cluster running this controller? on each runner host?

Reply:
single node cluster